### DISCUSSION TITLE
automatiom DRAM SI simulation using electronic desktop and PyAEDT

### DISCUSSION THREAD

**Discussion by mahakoraichi** [ansys/pyaedt#1820] [Q&A]
Hello,
I´m working on an automation project for DRAM SI simulation using electronic Desktop but unfortunatly I´m messing some commands to go forward with my script:

1. how I can import the S parameters and spice models?  I found just the one for ibis models and it´s get_ibis_model_from_file.
2. The command that set the IBIS parameters like bit pattern, trise, unit intervall( I found just the one to plot the eye diagram)
3. The function that detect the crossing point of two signals and the command that will calculate the distance from this point to other signals.
4. I found just the command that call the ibis buffer but the IBIS pin I didn´t find.
´
Can anyone help me please?
Thank you so much
Best Regards
Maha Koraichi

**Comment by Samuelopez-ansys** [comment]
Hi @mahakoraichi ,

Did you find the solution to your question?

Please change to Ansys developer portal to submit them, and please ask 1 by 1 because they are different and the answer will be faster.

https://discuss.ansys.com/
